packages/orbit-components/src/CallOutBanner/index.tsx

```diff
@@ -67,25 +18,36 @@ const CallOutBanner = ({
   description,
   dataTest,
   id,
-}: Props) => (
-  <StyledCallOutBanner
-    onClick={onClick}
-    tabIndex={(onClick || Number(tabIndex)) && (Number(tabIndex) || 0)}
-    data-test={dataTest}
-    id={id}
-  >
-    {illustration && <StyledIllustration>{illustration}</StyledIllustration>}
-    <Stack spacing="medium">
-      <Stack spacing="XSmall">
-        <Stack spacing="XXSmall">
-          {title && <Heading type="title3">{title}</Heading>}
-          {description && <Text>{description}</Text>}
+}: Props) => {
+  const Element = onClick ? "button" : "div";
```

Comment:
This is a good accessibility thing and the actual correct thing to do, but the button does not grow like a div. Do you think adding `w-full` is harmless here? It stretched the element to the full width but we did not have this property before, so I don't really know if it might break existing layouts 🤔 

**This branch:**
<img width="1330" alt="image" src="https://github.com/kiwicom/orbit/assets/6265045/da2d958e-a6e8-4b62-9a8f-e988e6f462f4">

**Master:**
<img width="1330" alt="image" src="https://github.com/kiwicom/orbit/assets/6265045/2c544fc5-32f7-42aa-91c9-0fb95ad94f29">

Reply:
Good catch! I’m not 100% sure it won’t break any existing layout. I think it will be better to get back to `div` for now to prevent any breaking changes. I’ll update the code.

Reply:
Updated 👍